README.md

```diff
@@ -1,15 +1,111 @@
+2. These are 10 cards in each color - red,green,yellow and blue and action cards are - 4 Draw One cards, 4 Skip cards, 2 Wild cards,
+   2 Wild Draw two cards.
+3. To be declared as the winner of the game, you have to be the first player to exhaust all the cards assigned to you.
+4. During the start of the game, 5 random cards are assigned to each player - human player and the computer, in this case.
+5. One card is placed on the table with which you have to start matching your cards.
+6. To keep things simple, the human player always starts the game first.
+7. These are the keyboard inputs with which human can play the game
+   * To request a card from the Draw pile, Press N or n.
+   * To end turn, Press E or e.
+   * To discard a normal card - Say you want to discard(RED,3), Press R3.
+   * To discard action cards, 
+     * Skip cards- Combine the first letter of the color with 'S', so if you want to discard(RED,SKIP), Press RS.
+     * Draw one cards- Combine the first letter of the color with 'O', so if you want to discard(RED,DRAW-ONE), Press R0.
+     * Wild cards- Press AW.
+     * Wild cards- Press AT.
+     (A above stands for Action cards, W stands for Wild, T stands for draw Two)
+   * To quit the game, Press Q or q. (This can happen at any point of time).
+
+
+## File Structure
+
+The following file structure is used to organize files in this repository:
+
+* doc - Includes the designed flowhcart for the application.  The main documentation can be found on the [UNO Express Card Game Documentation](https://github.com/MiaYWei/Group_G_UNO-Card-Game/wiki/User-Manual).
+* include - The main header files are listed below:
+  *  All Header files are included here that are used throughout the application.
+* src - Contains the definition for all functions.
+* test
+  * include - Contains the declarations for all unit tests.
+    * Includes the header files for the test files.       
+  * src - Contains the definitions for all unit tests.  
+    * test.c - Main file that calls the test functions in other files.
+
+## Installation
+
+Complete instructions regarding installation of this project can be found here [UNO Express Card Game User Documentation](https://github.com/MiaYWei/Group_G_UNO-Card-Game/wiki/User-Manual).
+
+### Compile and Build Instructions
+
+The following make commands are available to help compile and build the application:
+
+* `make all` (default) Will build the application and test application.
+
+### Run Application
+
+To run the application after installation, execute the application in the `bin` directory with  the command `UNO-Card-Game.exe`.
+
+
+Additional details can be found in the [user and developer documentation](https://github.com/MiaYWei/Group_G_UNO-Card-Game/wiki).
+
+### Test Application
+
+To run the test files, execute the application in the `bin` directory with `UNO-Card-Game_Test`. All test results are logged to the file 'test_log' in the same directory.
+
```

Comment:
"All test results are logged to the file 'test_log' in the same directory." - just currently. I think we need to relocate it to test-data folder. I will make the change today.

Reply:
Yes, we can make the change later when you're done with the modifications